src/applications/mhv-supply-reordering/mocks/feature-toggles/index.js

```diff
@@ -0,0 +1,33 @@
+const { snakeCase } = require('lodash');
```

Comment:
This is a neat approach but I don't love the lodash dependency since it's pretty heavy. (maybe it doesn't matter if it gets tree shaken?)